.github/workflows/check-users-in-space-developer-role.yml

```
@@ -0,0 +1,29 @@
+name: 'checker users space developer role'
+
+on:
+  schedule: # 1700hrs Mon-Fri
+    - cron: '00 17 * * 1-5'
+
+jobs:
+  CHECK-SPACE-USER:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: DFE-Digital/github-actions/setup-cf-cli@master
+        with:
+          CF_USERNAME: ${{ secrets.CF_USER_PROD }}
+          CF_PASSWORD: ${{ secrets.CF_PASSWORD_PROD }}
+          CF_SPACE_NAME: bat-prod
+          CF_ORG_NAME: dfe
+          CF_API_URL:  https://api.london.cloud.service.gov.uk
+          INSTALL_CONDUIT: false
+
+
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Run powershell script
+        shell: pwsh
+        run: |
+          ./scripts/check-users-in-space-developer-role.ps1 ${{ env.CF_SPACE_NAME }} ${{secrets.SLACK_WEBHOOK}}
```

Maybe more out of paranoia than anything else, but since this is likely an sh command:


```suggestion
          ./scripts/check-users-in-space-developer-role.ps1 "${{ env.CF_SPACE_NAME }}" "${{secrets.SLACK_WEBHOOK}}"
```